strawberryfields/engine.py

```diff
@@ -317,10 +317,6 @@ class LocalEngine(BaseEngine):
         backend_options (None, Dict[str, Any]): keyword arguments to be passed to the backend
     """
 
-    def __init__(self, backend, *, backend_options=None):
-        backend_options = backend_options or {}
-        super().__init__(backend, backend_options)
-
```

Comment:
Curious why this could be moved to `BaseEngine`

Reply:
Ah, I  noticed that this `__init__` wasn't doing anything that `BaseEngine.__init__` wasn't doing, and the linter was complaining! Simply deleting this here caused everything to continue working fine.

Note that we plan to merge `BaseEngine` back with `LocalEngine`, since they are essentially the same class, it is the only inheritance, and we don't expect to ever have any other subclasses.